.travis.yml

```diff
@@ -13,7 +13,6 @@ cache:
 
 env:
   # we recommend testing LTS's and latest stable release (bonus points to beta/canary)
-  - EMBER_TRY_SCENARIO=ember-1.13.13
```

Comment:
TODO: consider removing the testing-checks for this now that it is no longer within the recommended LTS window